src/igvfd/schemas/changelogs/configuration_file.md

```diff
@@ -6,6 +6,7 @@
 * Extend `collections` enum list to include `IGVF_catalog_beta_v0.2`.
 * Extend `collections` enum list to include `IGVF_catalog_beta_v0.3`.
 * Extend `collections` enum list to include `IGVF_catalog_beta_v0.4`.
+* Add `analysis_step_version`.
```

Comment:
New change should be at the top of the section (before line 5 here)

Reply:
done